packages/presentation/lib/utils/images_container.dart

```diff
@@ -0,0 +1,28 @@
+class Assets {
```

Comment:
assets is not only images. it can be fonts, files too. so rename please to AssetsImages